common/src/BasePlugin.php

```diff
@@ -98,4 +98,12 @@ public function getPluginFile() {
         $reflect = new ReflectionClass($this);
         return dirname(dirname($reflect->getFilename())) . '/' . $this->getSlug() . '.php';
     }
+
+    public function getGtinMetaKey(): string {
+        return "_{$this->getOptionName('gtin')}";
+    }
+
+    public function getReviewsHook(): string {
+        return "_{$this->getOptionName('reviews_cron')}";
```

Comment:
Hook names don't generally start with an underscore. Also, if you're only using this in a single class (WooCommerce), better define it there.

Reply:
Why is this still here.  Looks like you meant to move it to `WooCommerce.php`?